tests/components/myuplink/test_config_flow.py

```diff
@@ -81,3 +81,31 @@ async def test_full_flow(
 
     assert len(hass.config_entries.async_entries(DOMAIN)) == 1
     assert len(mock_setup.mock_calls) == 1
+
+
+async def test_flow_reauth(
+    hass: HomeAssistant,
+    init_integration,
+) -> None:
+    """Test reauth step."""
+    # entry = mock_config_entry(hass)
+    entry = init_integration
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={
+            "source": config_entries.SOURCE_REAUTH,
+            "entry_id": entry.entry_id,
+            # "unique_id": entry.unique_id,
+        },
+        data=entry.data,
```

Comment:
Passing the existing config entry data is correct. Don't do that like the example I posted. :smile: